util/src/time.cc

```diff
@@ -0,0 +1,65 @@
+#include <boost/date_time/posix_time/posix_time.hpp>
+#include <leatherman/util/time.hpp>
+
+namespace leatherman { namespace util {
+
+    std::string getExpiryDatetime(int expiry_minutes) {
+        struct tm expiry_time_info;
+        std::string expiry_time_buffer(80, '\0');
+
+        // Get current time and add the specified minutes
+        time_t expiry_time { time(nullptr) };
+        expiry_time += 60 * expiry_minutes;
+
+        // Get local time structure
+        get_local_time(&expiry_time, &expiry_time_info);
+
+        // Return the formatted string
+        if (strftime(&expiry_time_buffer[0], 80, "%FT%TZ", &expiry_time_info) == 0) {
+            // invalid buffer
+            return "";
+        }
+
+        expiry_time_buffer.resize(strlen(&expiry_time_buffer[0]));
+        return expiry_time_buffer;
+    }
+
+    void displayProgress(double percent, int len, std::string status) {
```

Comment:
@parisiale you added displayProgress in another project without using it. Was this something you expected to use in the future, or an accident?